packages/synthetics-sdk-broken-links/src/options_func.ts

```diff
@@ -203,17 +244,26 @@ export function setDefaultOptions(
     wait_for_selector: '',
     per_link_options: {},
     total_synthetic_timeout_millis: 60000,
+    screenshot_options: {
+      screenshot_condition:
+        BrokenLinksResultV1_BrokenLinkCheckerOptions_ScreenshotOptions_ScreenshotCondition.FAILING,
+      storage_location: '',
```

Comment:
the storage_location in the passed options won't get updated since the user did not specify it (we are just retruning what they gave us).

In general, if we receive an empty string we will create a bucket called gcm-<project-id>-synthetics-<cloud-region>. This will be passed to the user in a field called exeuction_data_storage_path (<bucket>/folders/.../<execution-data>/)

The logic to create the bucket is in following PRs